_rules/links-with-identical-names-and-context-serve-equivalent-purpose-fd3a94.md

```diff
@@ -229,80 +234,124 @@ These two HTML `a` elements have the same [accessible name][] and [context][prog
+
+```html
+<html lang="en">
+	<p>
+		<h2>Contact us:</h2>
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/contact-us.html?page=1" style="display:inline-block; background: url(/test-assets/shared/chat.png) 0 / 40px no-repeat; padding: 20px 0 20px 50px;">Contact Us</a>
+		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/contact-us.html?page=2" style="display:inline-block; background: url(/test-assets/shared/phone.png) 0 / 40px no-repeat; padding: 20px 0 20px 50px; margin-left: 40px;">Contact Us</a>
+	</p>
+</html>
+```
+
 #### Failed Example 2
 
-These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context]. They link to web pages that are similar, but have different information in their content.
+These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context], but go to different resources.
 
 ```html
 <html lang="en">
 	<div>
-		Learn more (<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html"
-			>Contact us</a
-		>) and get in touch (
-		<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html"
-			>Contact us</a
-		>)
+		<span style="text-align:center;">Contact us</span>
+		<span style="display:flex; justify-content:space-around;">
+			<img src="/test-assets/shared/chat.png" alt="Chat" style="width:50%;">
+			<img src="/test-assets/shared/phone.png" alt="Phone" style="width:50%;">
+		</span>
+		<span style="display:flex; justify-content:space-around;">
+			<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/contact-us.html?page=60e950cff70bf1ec60a702086748ab4dec361514">Contact Us</a>
+			<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/contact-us.html?page=c1d4e0f067462f4b28716f028d9213a25eb82f28">Contact Us</a>
+		</span>
 	</div>
 </html>
 ```
 
 #### Failed Example 3
 
+These two HTML `a` elements have the same [accessible name][] and [context][programmatically determined link context]. They link to web pages that are similar, but have different information in their content.
+
+```html
+<html lang="en">
+	<div>
+		<span style="text-align:center;">Contact us</span>
+		<span style="display:flex; justify-content:space-around;">
+			<img src="/test-assets/shared/chat.png" alt="Chat" style="width:50%;">
+			<img src="/test-assets/shared/phone.png" alt="Phone" style="width:50%;">
+		</span>
+		<span style="display:flex; justify-content:space-around;">
+			<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/about/contact.html">Contact Us</a>
+			<a href="/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/admissions/contact.html">Contact Us</a>
+		</span>
```

Comment:
I'm not sure the chat/phone disambiguation is good enough given the links.
That is, as a sighted user, how would I know that "Contact us (phone)" is the link to click to access the Admission office rather than general reception?

Reply:
I've added 2 blocks in _/test-assets/links-with-identical-names-serve-equivalent-purpose-b20e66/contact-us.html_ file to manage this scenario. I've created similar breadcrumbs, similar services navigation, similar headings but different content, to reflect the failure description.